### PR TITLE
tectonic: add page

### PR DESCRIPTION
**Pull request by seadanda**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** 0.14.1

### CODE REVIEW

**Comment by seadanda** [comment]
Apologies for the noise, my old work email was associated with the commit. I've reset the author and force pushed to fix it.

File: pages/common/tectonic.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# tectonic
+
+> A modern, self-contained TeX/LaTeX engine.
+> More information: <https://github.com/tectonic-typesetting/tectonic>.
+
+- Compile a standalone TeX/LaTeX file:
+
+`tectonic -X compile {{file.tex}}`
```
Comment (blueskyson) [inline pages/common/tectonic.md:8]:
```suggestion
`tectonic -X compile {{path/to/file.tex}}`
```

File: pages/common/tectonic.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# tectonic
+
+> A modern, self-contained TeX/LaTeX engine.
+> More information: <https://github.com/tectonic-typesetting/tectonic>.
```
Comment (blueskyson) [inline pages/common/tectonic.md:4]: I think this page is more informative.
```suggestion
> More information: <https://tectonic-typesetting.github.io/book/latest>.
```

File: pages/common/tectonic.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Compile a standalone TeX/LaTeX file:
+
+`tectonic -X compile {{file.tex}}`
+
+- Compile standalone with synctex data:
+
+`tectonic -X compile --synctex {{file.tex}}`
```
Comment (blueskyson) [inline pages/common/tectonic.md:12]:
```suggestion
`tectonic -X compile --synctex {{path/to/file.tex}}`
```

**Review by blueskyson — COMMENTED** [review]
> Apologies for the noise, my old work email was associated with the commit. I've reset the author and force pushed to fix it.

No worry. It's normal for new contributors.

**Comment by seadanda** [comment]
Thanks for the suggestions @blueskyson, I agree that they improve the listing and have committed the changes.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution and welcome to tldr.

File: pages/common/tectonic.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+- Initialize a tectonic project in the current directory:
+
+`tectonic -X init`
+
+- Create a tectonic project in the specified directory:
+
+`tectonic -X new {{project_name}}`
+
+- Build the existing project in the current directory:
```
Comment (acuteenvy) [inline pages/common/tectonic.md:22]:
```suggestion
- Build the project in the current directory:
```

File: pages/common/tectonic.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+# tectonic
+
+> A modern, self-contained TeX/LaTeX engine.
+> More information: <https://tectonic-typesetting.github.io/book/latest>.
+
+- Compile a standalone TeX/LaTeX file:
+
+`tectonic -X compile {{path/to/file.tex}}`
+
+- Compile standalone with synctex data:
```
Comment (acuteenvy) [inline pages/common/tectonic.md:10]:
```suggestion
- Compile a standalone TeX/LaTeX file with synctex data:
```

File: pages/common/tectonic.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+`tectonic -X compile --synctex {{path/to/file.tex}}`
+
+- Initialize a tectonic project in the current directory:
+
+`tectonic -X init`
+
+- Create a tectonic project in the specified directory:
```
Comment (acuteenvy) [inline pages/common/tectonic.md:18]:
```suggestion
- Initialize a tectonic project in the specified directory:
```

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**